src/Microsoft.IdentityModel.Tokens/Validation/Validators.IssuerSigningKey.cs

```diff
@@ -44,7 +45,7 @@ public static partial class Validators
         /// <exception cref="ArgumentNullException"> if 'securityKey' is null and ValidateIssuerSigningKey is true.</exception>
         /// <exception cref="ArgumentNullException"> if 'securityToken' is null and ValidateIssuerSigningKey is true.</exception>
         /// <exception cref="ArgumentNullException"> if 'validationParameters' is null.</exception>
```

Comment:
Will you be updating the comments on this PR?

Reply:
Answered in a related comment.